tests/extensions/test_check_docs.py

```diff
@@ -613,25 +610,6 @@ def my_func(named_arg, **kwargs):
         "tuple(int or str) or list(int or str)",
     ]
 
-    @pytest.mark.parametrize("complex_type", COMPLEX_TYPES)
```

Comment:
Maybe keeping the parametrize make sense here ? Good thing we can revert this commit in particular if you agree

Reply:
I don't think we should keep tests in two places. I have merged them in:
https://github.com/PyCQA/pylint/runs/4445138080?check_suite_focus=true

Reply:
> I don't think we should keep tests in two places.

Fair enough, I asked because the parametrize was quite elegant imo and I think removed a lot of duplication.

Reply:
Your solution is very elegant :+1:

Reply:
😄